_invokers/command.md

```diff
@@ -12,4 +12,121 @@ releases: [
 ]
 ---
 
-<!-- TODO write something meaningful -->
+The command function invoker provides a Docker base layer for a function consisting of a single command.
+It accepts gRPC requests, invokes the command for each request in the input stream,
+and sends the command's output to the stream of gRPC responses.
+
+## Install as a riff invoker
+
+```bash
+riff invokers apply -f https://github.com/projectriff/command-function-invoker/raw/master/command-invoker.yaml
+```
+
+## riff Commands
```

Comment:
I'm hesitant to include the riff-cli docs unless we also have a mechanism to keep them up to date.

Reply:
ok - agree - will remove.